src/mod_version.erl

```diff
@@ -22,6 +23,14 @@ stop(Host) ->
     mod_disco:unregister_feature(Host, ?NS_VERSION),
     gen_iq_handler:remove_iq_handler(ejabberd_local, Host, ?NS_VERSION).
 
+-spec config_spec() -> mongoose_config_spec:config_section().
+config_spec() ->
+    #section{
+       items = #{<<"iqdisc">> => mongoose_config_spec:iqdisc(),
```

Comment:
The docs say: `Default: "no_queue"`
The code says: `IQDisc = gen_mod:get_opt(iqdisc, Opts, one_queue),`
Could you fix the docs?